EIPS/eip-5170.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 5170
+title: Asserting authenticity of Client Script for Token Contracts
```

Comment:
```suggestion
title: Authenticity Proofs for Token Client Scripts
```
Required: The title is too long. The title I have here is exactly 44 characters (the maximum)